libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -288,6 +296,13 @@ dictionary ReverseSwapPairInfo {
     u64 fees_claim;
 };
 
+dictionary SimpleReverseSwapInfo {
```

Comment:
SimpleReverseSwapInfo is not a good name for the end user.
Better to use ReverseSwapInfo or ReverseSwap

Reply:
Done, renamed
- `ReverseSwapInfo` => `FullReverseSwapInfo`
- `SimpleReverseSwapInfo` => `ReverseSwapInfo`